src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseFactory.java

```diff
@@ -26,35 +26,78 @@ public class ProgrammingExerciseFactory {
 
     public static final String DEFAULT_BRANCH = "main";
 
+    /**
+     * Generates a programming exercise with the given release and due date. This exercise is added to the passed course.
```

Comment:
```suggestion
     * Generates a programming exercise with the given release and due date. This exercise is added to the provided course.
```

I suggest changing this for all occurrences.

Reply:
done